CHANGELOG.md

```diff
@@ -9,10 +9,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
-- Upgraded upstream chart from 3.2.1 to 3.10.0 - see [changelog](https://github.com/grafana/loki/blob/main/production/helm/loki/CHANGELOG.md) for more information.
-- upgraded Loki from 2.6.1 to 2.7.0 - see [changelog](https://github.com/grafana/loki/blob/main/CHANGELOG.md) for more information.
+- Upgraded upstream chart from 3.2.1 to 4.4.2 - see [changelog](https://github.com/grafana/loki/blob/main/production/helm/loki/CHANGELOG.md) for more information.
+- upgraded Loki from 2.6.1 to 2.7.2 - see [changelog](https://github.com/grafana/loki/blob/main/CHANGELOG.md) for more information.
 - ⚠️  [breaking changes](https://github.com/giantswarm/loki-app/blob/master/README.md#from-05x-to-06x)
   - nginx file definition for loki-multi-tenant has moved to a helper template.
+- sample configs: updated schema version to v12
```

Comment:
What is the implication of this ?

Reply:
1- this is a sample config. It just shows what a user config looks like.
2- should we want to change this for existing deployments of the app, this requires to take some precautions if we don't want to lose data.

I added a link to the doc in the sample configs that explain how to upgrade this.

Reply:
I think he might have meant what does it mean to change to v12 :)

Reply:
oh, well, there is a bit of doc here: https://grafana.com/blog/2022/04/11/new-in-grafana-loki-2.5-faster-queries-more-log-sources-so-long-s3-rate-limits-and-more/
Basically dealing better with S3 limits, leading to performance improvements.

Should this be documented in our sample configs, or it's just for your knowledge and then "that's the latest version, latest == best"?

Reply:
I'm wondering if we should not link this here so customers can read about it but I'm fine with how it is still :)

Reply:
ok ok, added the link :wink:

Reply:
You basically answered my question. Having the link and stating performance improvements in the changelog would have been nice too.